pkg/label/app.go

```diff
@@ -3,4 +3,7 @@ package label
 const (
 	// App is a standard label for tenant resources.
 	App = "app"
+
+	// AppOperatorHelmMajorVersion is a label for chart-operator app CRs.
+	AppOperatorHelmMajorVersion = "app-operator.giantswarm.io/helm-major-version"
```

Comment:
This is the new label. It controls which instance of app-operator bootstraps chart-operator based on the helm version.